index.html

```diff
@@ -25,14 +26,14 @@ <h2 class="card__question">
             <button class="card__button-answer" type="button">
               Show answer
             </button>
-            <p class="card__answer">Mark Hamill</p>
+            <p class="card__answer" data-js="answerOne">Mark Hamill</p>
             <ul class="card__tag-list">
               <li class="card__tag-list-item">#kingdomheart</li>
               <li class="card__tag-list-item">#mastereraqus</li>
               <li class="card__tag-list-item">#english voices</li>
             </ul>
             <div class="card__button-bookmark">
-              <button class="bookmark" aria-label="bookmark" type="button">
+              <button class="bookmark" aria-label="bookmark" type="button" data-js="bookMark">
```

Comment:
```suggestion
              <button class="bookmark" aria-label="bookmark" type="button" data-js="bookmark">
```
Same here, no camelcase needed for that